apps/api/prisma/schema.prisma

```diff
@@ -75,6 +78,20 @@ model OrganizationInvite {
   @@unique([inviteeEmail, organizationId])
 }
 
+model Events {
+  id              String  @id @unique @default(dbgenerated("(gen_random_uuid())::text"))
+  name            String
+  description     String
+  website         String?
+  location        String
+  createdAt       DateTime @default(now())
+  lastDate        DateTime
+  organizationId  String
```

Comment:
i was wondering what if the event is hosted virtually  ? 
can we add isOffline boolean if the event is offline we will show the location input in front end else we wont , 
thoughts  @DarkPhoenix2704  @reenphygeorge  ?

Reply:
what about making location as 'remote' for such events ?

Reply:
location is enough. 
If it is online, people can set it as Remote as said by @reenphygeorge